_includes/header.html

```diff
@@ -2,7 +2,7 @@
 <!--BEGIN TITLEAREA-->
 <div id="titlearea">
  <img id="projectlogo" alt="OpenOrienteering" src="{{ "/openorienteering.png" | prepend: site.baseurl }}"/><!--
- --><span id="projectname">{{ site.project_name | replace: " ", "&nbsp;" }}&nbsp;{{ version }}</span>
+ --><h1 id="projectname">{{ site.project_name | replace: " ", "&nbsp;" }}&nbsp;{{ version }}</h1>
```

Comment:
Then I think we should remove this string, a `span` element does not make sense ...